content/sql/shop-project-part-2/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+content_type: project
+flavours:
+- none
+prerequisites:
+  hard:
+  - sql/shop-project
+  soft: []
+story_points: 3
+from_repo: sql/shop-project
+submission_type: continue_repo
+tags:
+- sql
+- skill/databases
+ready: false
+title: Shop Database using sql - part 2
+---
+
+In the last part of this project you created a Shop Database and performed some SQL queries, now it is time to add more queries.
+
+## Instructions
+
+Create a new branch and ensure your repository maintains the same structure with additional new queries.
+
+### Querying a database
+
+Save all of your instructions in script files.
```

Comment:
- This is a bit confusing, what instructions?
- I think it would be better to mention that the code for the following questions should go into the query script from the previous part 1